x-pack/plugins/infra/public/components/logging/log_text_stream/log_entry_field_column.tsx

```diff
@@ -7,27 +7,52 @@
 import { darken, transparentize } from 'polished';
 import React, { useMemo } from 'react';
 
-import { css } from '../../../../../../common/eui_styled_components';
+import styled, { css } from '../../../../../../common/eui_styled_components';
 import { LogEntryColumnContent } from './log_entry_column';
 
 interface LogEntryFieldColumnProps {
   encodedValue: string;
   isHighlighted: boolean;
   isHovered: boolean;
   isWrapped: boolean;
+  isEnzyme?: boolean;
 }
 
+const CommaSeparatedLI = styled.li`
+  display: inline;
+  &:not(:last-child) {
+    margin-right: 1ex;
+  }
+`;
+
 export const LogEntryFieldColumn: React.FunctionComponent<LogEntryFieldColumnProps> = ({
   encodedValue,
   isHighlighted,
   isHovered,
   isWrapped,
+  isEnzyme,
 }) => {
   const value = useMemo(() => JSON.parse(encodedValue), [encodedValue]);
-
+  const formattedValue = Array.isArray(value) ? (
+    <ul>
+      {value.map((entry, i) => (
+        <CommaSeparatedLI key={`LogEntryFieldColumn-${i}`}>
+          {entry}
+          {i < value.length - 1 && ','}
```

Comment:
Ahh, that's why that didn't work when I tried it. I accidentally did `&:after` instead of `&::after`.

I wonder if there's a linter for styled-components that would catch things like that

Reply:
There might be, but our styled-components usage will soon come to an end anyway if I understand the design team's plans correctly.